### DISCUSSION TITLE
Change tools icon in Basic App Layout

### DISCUSSION THREAD

**Discussion by rabalasubramani3** [cloudscape-design/components#2177] [Q&A] [answered]
How can I change icon (the info icon) on tools in Basic App Layout? I want to use something other than the "info" icon.

**Comment by mv-anghiem** [comment]
Is there any update on this? I’d love to change the info icon. Thanks!

**Reply by just-boris** [reply, marked as answer]
Answered here: https://github.com/cloudscape-design/components/issues/1891
